.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "pwa-chrome",
+      "request": "launch",
+      "name": "Launch Chrome against Home Assistant",
+      "url": "http://localhost:48123",
+      "webRoot": "${workspaceFolder}/"
```

Comment:
I'm not sure how helpful this file is. It causes VS Code to enter attach to Chrome in debug mode, while there is nothing to debug in this situation.

We should definitely be able to somehow hook the Python debugger into it in the future though.

```suggestion
      "resolveSourceMapLocations": ["!**"]
```

This will ensure the VS Code won't try to interpret the existing source maps which HA frontend provides (it makes the browser laggy).